.gitignore

```diff
@@ -1,4 +1,5 @@
 *~
 report_outputs
+reportfactory.Rproj
```

Comment:
There is no reason to ignore the Rproj file itself. It's useful for new contributors. 

```suggestion
```